pkg/config/config.go

```diff
@@ -112,6 +120,30 @@ func (c *ConfigInfo) writeToFile() error {
 	return nil
 }
 
+// SetConfiguration modifies Odo configurations in the config file
+// as of now only being used for updatenotification
+func (c *ConfigInfo) SetConfiguration(parameter string, value bool) error {
+	switch parameter {
+	case "updatenotification":
+		c.OdoSettings.UpdateNotification = &value
+	default:
+		errors.Errorf("Unknown parameter :'%s' is not a parameter in odo config\nfor help run : odo config --help\n", parameter)
```

Comment:
Go errors shouldn't start with a capital letter. - https://github.com/golang/go/wiki/Errors

Also there shouldn't be any new lines.